api/proto/teleport/presence/v1/service.proto

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.presence.v1;
+
+import "google/protobuf/empty.proto";
+import "teleport/legacy/types/types.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/presence/v1;presencev1";
+
+// PresenceService provides methods to manage presence of RemoteClusters
+service PresenceService {
```

Comment:
I've omitted Upsert as we don't currently have that on the HTTP side, open to arguments that I should include it.

Reply:
The resource guideline also suggests to prefer Create and Update over Upsert, so this should be fine? - https://github.com/gravitational/teleport/blob/master/rfd/0153-resource-guidelines.md#upsert

Reply:
> I don't know that we should be exposing a Create/Update/Upsert for a remote cluster. They are intended to only be created internally by Auth during the validation process of adding a trusted cluster to the leaf. From what I can tell the existing REST API only exposes Create but I can't find any usages of that.

We also expose an `UpdateRemoteCluster` on the big auth server gRPC API - it looks like this is exposed in `tctl`.

Reply:
More than happy to drop the Create RPC from this and just delete the HTTP endpoint/client methods - less work for me!

Reply:
c4e25b1527